chapter_4/ex05_consumer/CMakeLists.txt

```
@@ -0,0 +1,41 @@
+# CMakeLists file for the Chapter 4 Example nr. 05: 
+# Example CMake consumer for Config-file package
+#
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.21)
+
+project(
+  ch4_ex05_consumer
+  VERSION 1.0
+  DESCRIPTION "Chapter 4 Example 05, Out ot project config-file package consumer"
+  LANGUAGES CXX)
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(FATAL_ERROR "The chapter-4, ex05_consumer project is intended to be a standalone, top-level project. Do not include this directory.")
```

```suggestion
  message(FATAL_ERROR "The chapter-4, ex05_consumer project is intended to be a standalone, top-level project. Do not vendor this directory.")
```

I could just be overly pedantic, but this implies the `include()` command and not the `add_subdirectory()` command otherwise.